bears/vcs/git/GitCommitBear.py

```diff
@@ -134,6 +145,34 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             body,
             **self.get_issue_checks_metadata().filter_parameters(kwargs))
 
+    def check_auth_email(self, auth_email,
+                         auth_email_check: bool=True,
+                         email_dns_check: bool=True):
+        """
+        Checks validity of author's email
+
+        :param auth_email_check:      If True, it will check validity of
+                                      author's email.
+        :param email_dns_check:       If True, it will confirm the domain
+                                      in author's email is valid.
+        """
+        if not auth_email_check:
+            return
+
+        # Used regex provided in http://emailregex.com/
+        email_check = re.match(r'(^[a-zA-Z0-9_.+-]'
```

Comment:
Regarding `validate_email`, that problem would go away if you disable its smtp validator, which is the appropriate setting anyway... but `is_email` does look to be more mature, with lots of tests, so good choice! :+1: